features/placeholder-shown.yml

```
@@ -0,0 +1,11 @@
+name: :placeholder-shown
+description: The `:placeholder-shown` CSS pseudo-element selects `<input>` and `<textarea>` elements when no value is set and placeholder text is shown.
```

Two ideas here:

- Avoid circularity with "…is shown"
- Distinguish with similarly-named pseudo?

```suggestion
description: The `:placeholder-shown` CSS pseudo-element selects `<input>` and `<textarea>` elements when no value is set and the element's `placeholder` attribute is not empty. Not to be confused with `::placeholder`, which selects the placeholder text itself.
```